docs/release-notes/NuGet-6.4.md

```diff
@@ -0,0 +1,115 @@
+---
+title: NuGet 6.4 Release Notes
+description: Release notes for NuGet 6.4 including new features, bug fixes, and DCRs.
+author: jebriede
+ms.author: jebriede
+ms.date: 10/27/2022
+ms.topic: conceptual
+---
+
+# NuGet 6.4 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**6.4**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.4](https://visualstudio.microsoft.com/downloads/) | [7.0.100](https://dotnet.microsoft.com/download/dotnet-core/7.0)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 17.4 with .NET Core workload
+
+## Summary: What's New in 6.4
+
+* Add ability to designate a package reference as used by every project in the repo - GlobalPackageReference [#10159](https://github.com/NuGet/Home/issues/10159)
+
+* Improved performance loading installed packages in the Package Manager UI - [#11923](https://github.com/NuGet/Home/issues/11923)
```

Comment:
This bug has improved performance of PM UI in general across all tabs and solution restore scenario.

Reply:
Updated to:
> Improved performance loading packages for all tabs in the Package Manager UI

PTAL